staging/src/k8s.io/client-go/rest/exec_test.go

```diff
@@ -242,6 +242,9 @@ func TestConfigToExecClusterRoundtrip(t *testing.T) {
 		func(h *WarningHandler, f fuzz.Continue) {
 			*h = &fakeWarningHandler{}
 		},
+		func(h *WarningHandlerWithContext, f fuzz.Continue) {
+			*h = &fakeWarningHandlerWithContext{}
```

Comment:
curious if these changes were required to pass tests or if this was just for tidyness

Reply:
oh, nevermind, a new field got added